api/data/network-capabilities.js

```diff
@@ -141,6 +141,23 @@ const getNetworkCapabilities = {
     action_vote: capabilityEnum.DISABLED,
     action_proposal: capabilityEnum.DISABLED
   },
+  'polkadot-testnet': {
+    feature_session: capabilityEnum.ENABLED,
+    feature_explore: capabilityEnum.ENABLED,
+    feature_portfolio: capabilityEnum.ENABLED,
+    feature_validators: capabilityEnum.ENABLED,
+    feature_proposals: capabilityEnum.DISABLED,
+    feature_activity: capabilityEnum.DISABLED,
+    feature_explorer: capabilityEnum.DISABLED,
+    action_send: capabilityEnum.ENABLED,
```

Comment:
are transfers enabled on polkadot yet?

Reply:
yes, they are. Actually this is outdated. Now even staking is enabled